blocks/header/header.js

```diff
@@ -104,23 +104,26 @@ function addHeaderInteractions(header) {
     };
   });
 
-  header.querySelectorAll('.drilldown__first-level-container li:has(ul) a').forEach((link) => {
-    link.onclick = (event) => {
-      event.preventDefault();
+  header.querySelectorAll('.drilldown__first-level-container li a').forEach((link) => {
+    // Can't use has operator as its not supported in firefox
+    if (link.nextElementSibling && link.nextElementSibling.tagName === 'UL') {
```

Comment:
I think the selector rule we have now is "handle clicks for all `.drilldown__first-level-container li a` where `<a>` is followed by a `<ul>`".
There earlier one was, "handle clicks for all `.drilldown__first-level-container li a` where `<li>` has a `<ul>` as its descendent. In effect, the `<ul>` may not be a sibling of `<a>` unlike what you proposed. 
With that said, the only place I see this being used is the megamenu 2'nd level drilldown and there the proposed rule works, so I think we should be ok.   
But I am not clear why this may cause the icons/images in the header to go missing.

Reply:
**addHeaderInteractions** function was preventing the code from calling **decorateIcons**. And that was leading to not loading icons.
Now, after David's fallback PR (https://github.com/hlxsites/zeiss/pull/225), **decorateIcons** is being called before **addHeaderInteractions**. So, that case will not occur. But still you can see in console that code is breaking in **addHeaderInteractions** function.